tijanabekic/assignment4/Assignment 4.java

```diff
@@ -0,0 +1,101 @@
+import static org.junit.Assert.*;
+
+public class A4Q1 {
+
+    public static void findIslandEnd(boolean[][] grid, int i, int j, int row, int column) {

```

Comment:
This method doesn't need to be public, as it is only called from inside findIslands.